### PR TITLE
Update infrared-openstack.sh 13 script

### PR DESCRIPTION
**Pull request by leifmadsen**
Remove the PREFIX option that I don't think we ever got working correctly. Import
the ENVIRONMENT_TEMPLATE variable that we use in the 16 script for future considerations.

Bump the base OS image and puddle target to match current phase 2 tests for OSP13.

Reduce CPU and memory footprint to match recent changes in 16 script.

### CODE REVIEW

**Comment by leifmadsen** [comment]
Going to run a test on this today.

**Comment by leifmadsen** [comment]
This is basically all working without issue now, except when you try to deploy with STF enabled. There appears to be some issue with container images syncing for collectd and qdrouterd as they do not appear to get pulled in.

File: tests/infrared/13/infrared-openstack.sh [hunk 1]
```diff
@@ -6,10 +6,11 @@ set -e
 #  ./infrared-openstack.sh
 
 VIRTHOST=${VIRTHOST:-localhost}
-AMQP_HOST=${AMQP_HOST:-stf-default-interconnect-5671-service-telemetry.apps-crc.testing}
+AMQP_HOST=${AMQP_HOST:-default-interconnect-5671-service-telemetry.apps.stf.cloudops.psi.redhat.com}
```
Comment (leifmadsen) [inline tests/infrared/13/infrared-openstack.sh:9]:
```suggestion
AMQP_HOST=${AMQP_HOST:-default-interconnect-5671-service-telemetry.apps-crc.testing}
```

**Comment by leifmadsen** [comment]
Completes installation at least. Will need to verify that I'm seeing telemetry data going to STF.

```
 Stack overcloud UPDATE_COMPLETE 

Started Mistral Workflow tripleo.deployment.v1.get_horizon_url. Execution ID: c7a84d77-daf6-4244-ad11-1463ec6b5b93
Overcloud Endpoint: http://10.0.0.132:5000/
Overcloud Horizon Dashboard URL: http://10.0.0.132:80/dashboard
Overcloud rc file: /home/stack/overcloudrc
Overcloud Deployed
```

As somewhat expected (not sure how to add additional images to the `container-images.yaml` and `docker-images.yaml` which is generated), the overcloud deployment fails when STF is enabled. This is easily worked around by adding the images to `container-images.yaml` on the undercloud and then syncing the changes to the local registry. Additional parameters are required in the `docker-images.yaml` to enable references to the proper images.

```
$ cat container_images.yaml | grep -E 'collectd|qdrouterd'
- imagename: registry-proxy.engineering.redhat.com/rh-osbs/rhosp13-openstack-collectd:20210324.1
- imagename: registry-proxy.engineering.redhat.com/rh-osbs/rhosp13-openstack-qdrouterd:13.0-67
```

```
$ cat virt/docker-images.yaml | grep -E 'collectd|qdrouterd'
  DockerCollectdConfigImage: 192.168.24.1:8787/rh-osbs/rhosp13-openstack-collectd:20210324.1
  DockerCollectdImage: 192.168.24.1:8787/rh-osbs/rhosp13-openstack-collectd:20210324.1
  DockerMetricsQdrConfigImage: 192.168.24.1:8787/rh-osbs/rhosp13-openstack-qdrouterd:13.0-67
  DockerMetricsQdrImage: 192.168.24.1:8787/rh-osbs/rhosp13-openstack-qdrouterd:13.0-67
  DockerQdrouterdConfigImage: 192.168.24.1:8787/rh-osbs/rhosp13-openstack-qdrouterd:13.0-67
  DockerQdrouterdImage: 192.168.24.1:8787/rh-osbs/rhosp13-openstack-qdrouterd:13.0-67
```

Sync the images from `container-images.yaml` to the internal registry:

```
openstack overcloud container image upload --verbose --config-file /home/stack/container_images.yaml
```

Then just run `overcloud_deploy.sh` again.

Multi-cloud enabled deployment done via:

```
CLOUD_NAME=cloudops-06 ENABLE_STF_CONNECTORS=true LIBVIRT_DISKPOOL="/home/libvirt/images" AMQP_HOST=default-interconnect-5671-service-telemetry.apps.stf.cloudops.psi.redhat.com ./infrared-openstack.sh
```

**Comment by leifmadsen** [comment]
This is ready for review and merge.

**Review by csibbitt — APPROVED** [review]
LGTM

**Comment by leifmadsen** [comment]
Just waiting for multi-cloud run to complete and verify that I'm getting data, and that it shows up via the appropriate topics. Then I will merge it.

**Pull request merged**